src/components/Restoration/YourCompanySummary.vue

```diff
@@ -61,7 +99,18 @@ export default {
     },
     getCourtOrderFileNumber () {
       return this.getRestoration.courtOrder.fileNumber
+    },
+    getCompanyName (): string {
+      if (this.getNameRequestLegalName) return this.getNameRequestLegalName
+      return `${this.getBusinessNumber || '[Incorporation Number]'} B.C. Ltd.`
+    },
```

Comment:
I would suggest this logic might be better located in the store because the logic behind the numbered company name can be more complex than `${this.getBusinessNumber || '[Incorporation Number]'} B.C. Ltd.` and it's logic that I'm guessing we'll want to reuse.  If I remember correctly, these are some examples:
- 0123456 B.C. Community Contribution Company
- 0123456 B.C. Ltd.
- 0123456 BC Unlimited Liability Company

Reply:
@jonathan-longe, I will keep a note of this. This affects almost all filings nd needs to be fixed. How about a techDebt ticket? Does this case qualify for techDebt?

Reply:
Sure, if it's difficult to change, then creating a tech debt ticket would be most appropriate.